pkg/pool-manager/pool.go

```diff
@@ -26,24 +26,25 @@ import (
 )
 
 const (
-	StartPoolRangeEnv  = "START_POOL_RANGE"
-	EndPoolRangeEnv    = "END_POOL_RANGE"
-	networksAnnotation = "k8s.v1.cni.cncf.io/networks"
+	StartPoolRangeEnv        = "START_POOL_RANGE"
+	EndPoolRangeEnv          = "END_POOL_RANGE"
+	networksAnnotation       = "k8s.v1.cni.cncf.io/networks"
+	networksStatusAnnotation = "k8s.v1.cni.cncf.io/networks-status"
 )
 
 var log = logf.Log.WithName("PoolManager")
 
 type PoolManager struct {
-	kubeClient      kubernetes.Interface // kubernetes client
-	startRange      net.HardwareAddr     // fist mac in range
-	endRange        net.HardwareAddr     // last mac in range
-	currentMac      net.HardwareAddr     // last given mac
-	macPoolMap      map[string]bool      // allocated mac map
-	podToMacPoolMap map[string][]string  // map for namespace/podname and a list of allocated mac addresses
-	vmToMacPoolMap  map[string][]string  // cap for namespace/vmname and a list of allocated mac addresses
-	poolMutex       sync.Mutex           // mutex for allocation an release
-	isLeader        bool                 // leader boolean
-	isKubevirt      bool                 // bool if kubevirt virtualmachine crd exist in the cluster
+	kubeClient      kubernetes.Interface         // kubernetes client
+	startRange      net.HardwareAddr             // fist mac in range
+	endRange        net.HardwareAddr             // last mac in range
+	currentMac      net.HardwareAddr             // last given mac
+	macPoolMap      map[string]bool              // allocated mac map
+	podToMacPoolMap map[string]map[string]string // map for namespace/podname and a mac of network name as key and mac address as value
```

Comment:
i don't understand the comment.

maybe 'map allocated mac address by networkname and namespace/podname: {"namespace/podname: {"network name": "mac address"}}'

Reply:
done